File: app/src/main/java/com.example.safehome/System/IUser.java
package System;

public interface IUser {

	string getName();

	/**
	 * 
	 * @param name
	 */
	void setName(string name);

	string getEmail();

	/**
	 * 
	 * @param email
	 */
	void setEmail(string email);

	List<Address> getAddresses();

	/**
	 * 
	 * @param addresses
	 */
	void setAddresses(List<Address> addresses);

	List<String> getSystemIDs();

	/**
	 * 
	 * @param systemIDs
	 */
	void setSystemIDs(List<String> systemIDs);

	List<Phone> getPhones();

	/**
	 * 
	 * @param phones
	 */
	void setPhones(List<Phone> phones);

	/**
	 * 
	 * @param addresses
	 */
	void setAddresses1(List<Address> addresses);

	/**
	 * 
	 * @param phones
	 */
	void setPhones1(List<Phone> phones);

	/**
	 * 
	 * @param addresses
	 */
	void setAddresses2(List<Address> addresses);

	/**
	 * 
	 * @param phones
	 */
	void setPhones2(List<Phone> phones);

	/**
	 * 
	 * @param addresses
	 */
	void setAddresses3(List<Address> addresses);

	/**
	 * 
	 * @param phones
	 */
	void setPhones4(List<Phone> phones);

}

File: app/src/main/kotlin/com.example.safehome/System/IUser.kt
package com.example.safehome.System

import android.location.Address
import android.provider.ContactsContract.CommonDataKinds.Phone

interface IUser {
    /**
     *
     * @param name
     */
    var name: String?

    /**
     *
     * @param email
     */
    var email: String?

    /**
     *
     * @param addresses
     */
    var addresses: List<Address?>?

    /**
     *
     * @param systemIDs
     */
    var systemIDs: List<String?>?

    /**
     *
     * @param phones
     */
    var phones: List<Phone?>?

    /**
     *
     * @param addresses
     */
    fun setAddresses1(addresses: List<Address?>?)

    /**
     *
     * @param phones
     */
    fun setPhones1(phones: List<Phone?>?)

    /**
     *
     * @param addresses
     */
    fun setAddresses2(addresses: List<Address?>?)

    /**
     *
     * @param phones
     */
    fun setPhones2(phones: List<Phone?>?)

    /**
     *
     * @param addresses
     */
    fun setAddresses3(addresses: List<Address?>?)

    /**
     *
     * @param phones
     */
    fun setPhones4(phones: List<Phone?>?)
}